synfig-studio/src/gui/trees/layertree.cpp

```diff
@@ -392,6 +397,39 @@ LayerTree::create_param_tree()
 	//column_time_track->set_visible(false);
 }
 
+bool
+LayerTree::search_param_tree(const Glib::RefPtr<Gtk::TreeModel>& model,int column,const Glib::ustring& search_string,const Gtk::TreeModel::iterator& iter)//make sure it is indeed search string
+{
+	Gtk::TreePath path(iter);
+	if(match_found && (path != match_path))// when match is found dont bother with others
+		return true;
+
+	Gtk::TreeModel::Row row = *iter;
+	Gtk::TreeModel::Children children = row.children();
+	Glib::ustring substr(search_string.uppercase());
+	Glib::ustring label((*iter)[param_model.label]);
+	label=label.uppercase();
+	if(label.find(substr)!=Glib::ustring::npos){//i.e. match found
+		match_found=false;
```

Comment:
comment and variable name/assignment diverge o.O

Reply:
xD, I don't really remember what I was thinking. Anyways this is remove also now.